### PR TITLE
Feature/footer

### PR DESCRIPTION
**Pull request by MatHenriquez**
# Sumary
- Create a **footer component** with links to the _Terms of Service_, _Privacy Policy_ and _GitHub repository_ for this application.
- **Render the footer** at the bottom of _home_ and _dashboards_ pages.

# Details
- Add a footer in the web application that can be used by the user to visit pages with legal information or the repository that stores the Stellar Wallet application.

# Evidences

## Footer in the Home Page
home_footer

## Footer tests passing
footer_tests

### CODE REVIEW

File: cypress/e2e/spec.cy.ts [hunk 1]
```diff
@@ -69,4 +69,64 @@ describe("End-to-End testing", () => {
...
+      cy.get('[data-cy="privacy-link"]').should("have.text", "Privacy Policy");
+      cy.get('[data-cy="repository-link"]').should("exist");
+    });
+
+    describe("Footer links", () => {
+      beforeEach(() => {
+        cy.visit("/");
+      });
+
+      it("Terms of service link should open the terms of service page in a new tab", () => {
```
Comment (Genebson) [inline cypress/e2e/spec.cy.ts]: `Should...`

File: cypress/e2e/spec.cy.ts [hunk 2]
```diff
@@ -69,4 +69,64 @@ describe("End-to-End testing", () => {
...
+        cy.get('[data-cy="terms-link"]')
+          .should("have.attr", "target", "blank")
+          .should("have.attr", "href", "https://stellar.org/terms-of-service");
+      });
+
+      it("Privacy policy link should open the the privacy policy page in a new tab", () => {
```
Comment (Genebson) [inline cypress/e2e/spec.cy.ts]: `Should...`

File: cypress/e2e/spec.cy.ts [hunk 3]
```diff
@@ -69,4 +69,64 @@ describe("End-to-End testing", () => {
...
+        cy.get('[data-cy="privacy-link"]')
+          .should("have.attr", "target", "blank")
+          .should("have.attr", "href", "https://stellar.org/privacy-policy");
+      });
+
+      it("Repository link should should open the repository page in a new tab", () => {
+        cy.get('[data-cy="repository-link"]')
```
Comment (Genebson) [inline cypress/e2e/spec.cy.ts:132]: `Should...`

File: app/components/Footer.tsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
+import { FC } from "react";
+
+const Footer: FC = () => {
+  const termsOfService: string = "https://stellar.org/terms-of-service";
+  const privacyPolicy: string = "https://stellar.org/privacy-policy";
+  const gitHubLink: string = "https://github.com/MatHenriquez/stellar-wallet";
```
Comment (Genebson) [inline app/components/Footer.tsx]: This should come from an environment variable.

**Review by Genebson — APPROVED** [review]

File: cypress/e2e/spec.cy.ts (hunk 1 above)
Comment (MatHenriquez) [reply cypress/e2e/spec.cy.ts]: I have rewritten this description to follow the format you indicated.

File: cypress/e2e/spec.cy.ts (hunk 2 above)
Comment (MatHenriquez) [reply cypress/e2e/spec.cy.ts]: I have rewritten this description to follow the format you indicated.

File: cypress/e2e/spec.cy.ts (hunk 3 above)
Comment (MatHenriquez) [reply cypress/e2e/spec.cy.ts:132]: I have already rewritten this description to follow the format you indicated.

File: app/components/Footer.tsx (hunk 4 above)
Comment (MatHenriquez) [reply app/components/Footer.tsx]: I have moved these links to the environment variables file and imported them from there

**Pull request merged**